### PR TITLE
[ACM-4102] Added cluster and policy group compliancy dashboards

### PR DESCRIPTION
**Pull request by dislbenn**
Adding global hub dashboards for cluster and policy group compliancy.

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml [hunk 1]
```diff
@@ -62,6 +62,10 @@ spec:
           name: grafana-dashboards
         - mountPath: /grafana-dashboards/0/acm-global-policy-overview
           name: grafana-dashboard-acm-global-policy-overview
+        - mountPath: /grafana-dashboards/0/acm-global-cluster-group-compliancy-overview
+          name: grafana-dashboard-acm-global-policy-cluster-compliancy-overview
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:66]: the name should be `grafana-dashboard-acm-global-cluster-group-compliancy-overview`

File: operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml [hunk 2]
```diff
@@ -136,6 +140,14 @@ spec:
           defaultMode: 420
           name: grafana-dashboard-acm-global-policy-overview
         name: grafana-dashboard-acm-global-policy-overview
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:142]: You may remove this configmap which is a initial one. And it is out of date.

Comment (dislbenn) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:142]: @clyang82 do we want to hold off on removing this dashboard, until the correct data tables are available for the newer dashboards?

Comment (clyang82) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/deployment.yaml:142]: that's fine.

**Comment by clyang82** [comment]
Let us have this PR merged. We need to update the query to use correct tables once it is ready.

/lgtm
/approved

**Pull request merged**